src/pkgmt/cli.py

```diff
@@ -174,3 +175,29 @@ def lint():
 
     if returncode:
         raise SystemExit("Error linting")
+
+
+@cli.command()
+@click.option(
```

Comment:
please move this under pkgmt/modified.py, so we can execute it with `python -m pkgmt.modified` (we don't want to add this to the main CLI since this is mostly for CI purposes)